src/main/app.ts

```diff
@@ -38,10 +38,11 @@ logger.info('Environment: ' + env);
 setupDev(app,developmentMode);
 setupTest(app);
 
+app.use(express.static(path.join(__dirname, 'public')));
 app.use(compression());
 app.use(favicon(path.join(__dirname, '/public/assets/images/favicon.ico')));
 app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+app.use(express.urlencoded({ extended: true}));
```

Comment:
According to the internet (https://stackoverflow.com/a/51844327), urlencoded is only for POST and PUT methods, and fetching static assets is, I think, always GET request, therefore this unlikely have anything to do with the issue.